MapleServer2/Packets/ChatPacket.cs

```diff
@@ -16,16 +19,23 @@ public static class ChatPacket {
                 .WriteByte()
                 .WriteInt(); // Channel
 
-            switch (type) {
+            switch (type)
+            {
                 case ChatType.WhisperFrom:
                     pWriter.WriteUnicodeString("???");
                     break;
                 case ChatType.Super:
-                    pWriter.WriteInt(); // item id?
+                    pWriter.WriteInt(20800017); // item id?
                     break;
                 case ChatType.UnknownPurple:
                     pWriter.WriteLong(); // char id?
                     break;
+                case ChatType.World:
+                    pWriter.WriteInt(); // Merets
```

Comment:
Was this tested? How do you know this is merets?

Reply:
True, is working for now, without sending anything, i was testing if it needs the merets in order to be send the message